src/test/resources/pages/instructorFeedbackAddSuccess.html

```diff
@@ -344,7 +344,7 @@ <h5 class="col-sm-4">
                                         </div>
                                     </div>
                                     <div class="row radio">
-                                        <div data-original-title="Select this option if you intend to manually publish the session later on." class="col-md-4" title="" data-toggle="tooltip" data-placement="top">
+                                        <div data-original-title="Select this option if you intend to manually publish the feedback responses for this session later on." class="col-md-4" title="" data-toggle="tooltip" data-placement="top">
```

Comment:
use 'responses' instead of 'feedback responses'. In the user's eye, not all sessions are 'feedback sessions' although in the code they are all considered `FeedbackSession`s. For the user, they can be surveys, quizzes, etc.